SmartDeviceLink-iOS/SmartDeviceLink/SDLLocalization.h

```diff
@@ -0,0 +1,83 @@
+// SDLLocalization.h
+//
+
+
+#import <Foundation/Foundation.h>
+
+/**
+ Provides a simplified way to handle localization for a specific language that may not match the system language.
+ */
+@interface SDLLocalization : NSObject
+
+/** The language code used by this instance. This code can differ from initialization regarding fallback. */
+@property (readonly) NSString *language;
+
+/** The region code used by this instance. This code can differ from initialization regarding fallback. */
+@property (readonly) NSString *region;
```

Comment:
Language and region code will come from SDL (see SDLLanguage). It will match ISO 639-1 and ISO 3166-1 (see https://developer.apple.com/library/ios/documentation/MacOSX/Conceptual/BPInternational/LanguageandLocaleIDs/LanguageandLocaleIDs.html)